atomic_reactor/schemas/config.json

```diff
@@ -584,22 +573,33 @@
                 "description": "Don't check SSL certificate for url",
                 "type": "boolean"
             },
-            "auth": {
-                "description": "Authentication information",
-                "type": "object",
-                   "properties": {
-                    "cfg_path": {
-                        "description": "Path to directory  containing .dockercfg for registry auth",
-                        "type": "string"
-                    }
-                },
-                "additionalProperties": false,
-                "required": ["cfg_path"]
-            }
+            "auth": {"$ref": "#/definitions/registry_auth"}
         },
         "additionalProperties": false,
         "required": ["url"]
     },
+    "pull_registries": {
+        "description": "Other container registries that may be used for pulling images",
+        "type": "array",
+        "items": {
+            "description": "Any container registry other than the default one",
+            "type": "object",
+            "properties": {
+                "hostname": {
+                    "description": "Registry hostname (not a URL, i.e. no http(s), no port)",
+                    "type": "string",
+                    "pattern": "^(?!http(s)?://).*$"
+                },
```

Comment:
@lkolacek @MartinBasti PTAL, do I understand correctly that it must be a plain hostname, not a URL? Or does it not really matter?

Reply:
yes hostname, but I'm checking the implementation and it seems that current code expects URLs. so we may need both?

Reply:
or having just URL and parsing hostname from that

Reply:
Ok, will use URL instead

Reply:
OK I will change it in design and PR in osbs-playbooks